src/js/views/globals/search/patient-search_views.js

```diff
@@ -116,16 +61,24 @@ const PatientSearchModal = View.extend({
       },
     });
 
-    if (search) this.collection.search(search);
+    if (search) this.collection.search(search, searchType);
```

Comment:
Pass the `searchType` to the patient search service. 

If `searchType === 'dob'`, an additional step is made to validate the inputted date before sending the api request.